networkit/__init__.py

```diff
@@ -134,7 +134,7 @@ def getComponentPartition(G):
 			cc = components.ConnectedComponents(G).run()
 		return cc.getPartition()
 
-	print("Network Properties for:\t\t{}".format(G.getName()))
+	print("Network Properties:")
```

Comment:
Is G.getName() dropped completely?

Reply:
Yes, I think that line is still file, the remaining network properties will be listed below.